### PR TITLE
[feature](merge-cloud) Add several multiple dimensions cloud metrics

### PR DESCRIPTION
**Pull request by swjtu-zhanglei**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/metric/CloudMetrics.java [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+    protected static AutoMappedMetric<GaugeMetricImpl<Double>> CLUSTER_QUERY_PER_SECOND_GAUGE;
+    protected static AutoMappedMetric<GaugeMetricImpl<Double>> CLUSTER_QUERY_ERR_RATE_GAUGE;
+
+    protected static AutoMappedMetric<Histogram> CLUSTER_QUERY_LATENCY_HISTO;
+
+    protected static AutoMappedMetric<GaugeMetricImpl<Integer>> CLUSTER_BACKEND_ALIVE;
+    protected static AutoMappedMetric<GaugeMetricImpl<Integer>> CLUSTER_BACKEND_ALIVE_TOTAL;
+
+    protected static void init() {
+        if (!Config.isCloudMode()) {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/metric/CloudMetrics.java:42]: Config.isNotCloudMode()

File: fe/fe-core/src/main/java/org/apache/doris/metric/MetricCalculator.java [hunk 2]
```diff
@@ -77,4 +87,69 @@ private void update() {
         }
         MetricRepo.GAUGE_MAX_TABLET_COMPACTION_SCORE.setValue(maxCompactionScore);
     }
+
+    private void initCloudMetrics() {
+        if (!Config.isCloudMode()) {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/metric/MetricCalculator.java:92]: ditto

**Review by gavinchou — COMMENTED** [review]
LGTM

**Comment by SWJTU-ZhangLei** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/metric/MetricCalculator.java (hunk 2 above)
Comment (SWJTU-ZhangLei) [reply fe/fe-core/src/main/java/org/apache/doris/metric/MetricCalculator.java:92]: > ditto
resolved

**Review by gavinchou — APPROVED** [review]

**Review by w41ter — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**